<launch>
  <!-- Requires Python2 - source relavent venv -->

  <arg name="recordData" default="false" />

  <node pkg="rl_pkg" name="hand_control" type="hand_control_sim.py" output="screen"></node>
  <node pkg="rl_pkg" name="runPG" type="runPG_sim.py" output="screen"/>

 <node pkg="rosbag" type="record" name="rosbag_record_data"
       args="record -O /home/pracsys/Dropbox/transfer/F  /gripper/load /gripper/pos /hand/obj_pos"
       if="$(arg recordData)" />

  <node pkg="rl_pkg" type="plot_obj_pos.py" name="Plot" output="screen"></node>

</launch>